<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AwEpochVisuWidgetUi</class>
 <widget class="QWidget" name="AwEpochVisuWidgetUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Review conditions</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources/AnyWave.qrc">
    <normaloff>:/images/AnyWave_icon.png</normaloff>:/images/AnyWave_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Conditions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxEpoch">
       <property name="title">
        <string>Epoch preview</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Condition:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCondition"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Mosaic View</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelEpoch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Epoch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonPrev">
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
            <property name="shortcut">
             <string>Left</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonNext">
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
            <property name="shortcut">
             <string>Right</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonReject">
            <property name="text">
             <string>Reject</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Averaging</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../resources/AnyWave.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AwEpochVisuWidgetUi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>AwEpochVisuWidgetUi</receiver>
   <slot>doAveraging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>582</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AwEpochVisuWidgetUi</receiver>
   <slot>openMosaicView()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>652</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>837</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doAveraging()</slot>
  <slot>openMosaicView()</slot>
 </slots>
</ui>
